resources/istio/README.md

```diff
@@ -53,26 +53,10 @@ The chart deploys Pods that consume minimum resources as specified in the resour
     $ kubectl create ns $NAMESPACE
     ```
 
-1. If you are enabling `kiali`, you need to create the secret that contains the username and passphrase for `kiali` dashboard:
-    ```
-    $ echo -n 'admin' | base64
-    YWRtaW4=
-    $ echo -n '1f2d1e2e67df' | base64
-    MWYyZDFlMmU2N2Rm
-    $ cat <<EOF | kubectl apply -f -
-    apiVersion: v1
-    kind: Secret
-    metadata:
-      name: kiali
-      namespace: $NAMESPACE
-      labels:
-        app: kiali
-    type: Opaque
-    data:
-      username: YWRtaW4=
-      passphrase: MWYyZDFlMmU2N2Rm
-    EOF
-    ```
+1. If you are enabling `kiali`, make sure you have enabled `prometheus-operator` and `monitoring` components. Kiali installer will generate the secret with the username `admin` and an autogenerated passphrase, you will need these to login to Kiali UI. To get the password use:
+```
+$ kubectl -n istio-system get secret kiali -o json | jq -r .data.passphrase | base64 -D | awk '{print "Password:", $1}' 
+```
```

Comment:
Additional comment - in the `Configuration` section please change the default value of the `global.tag` parameter. I checked with @strekm and it's `1.1.0`